### PR TITLE
Add conquer city quest for OCC games

### PR DESCRIPTION
**Pull request by axatin**
Fix for #8850 and #9028 

- The bugged CS quest to acquire a city is turned off for OCC games
- A new quest to destroy a city is added for OCC games instead
- The liberation quest can now also be given in OCC games

### CODE REVIEW

**Comment by IanE9** [comment]
Instead of adding a new quest type specifically for OOC games would it not be more appropriate to instead make `MINOR_CIV_QUEST_UNIT_GET_CITY` achievable by destroying the target city as you've done with the new quest type? Apologies if this has already been discussed somewhere and I've missed it.

File: CvGameCoreDLL_Expansion2/CvMinorCivAI.cpp [hunk 1]
```diff
@@ -2824,6 +2854,33 @@ void CvMinorCivQuest::DoStartQuest(int iStartTurn, PlayerTypes pCallingPlayer)
...
+
+	const char* strTargetNameKey = pCity->getNameKey();
+
+	strMessage = Localization::Lookup("TXT_KEY_NOTIFICATION_QUEST_UNIT_GET_CITY_OCC");
+	strMessage << strTargetNameKey;
+	strSummary = Localization::Lookup("TXT_KEY_NOTIFICATION_SUMMARY_UNIT_GET_CITY_OCC");
+	strSummary << strTargetNameKey;
+
+	iNotificationX = pCity->plot()->getX();
+	iNotificationY = pCity->plot()->getY();
```
Comment (IanE9) [inline CvGameCoreDLL_Expansion2/CvMinorCivAI.cpp:2882]: Loose indentation here. Empty line at the beginning of the else if body should also be removed.

File: CvGameCoreDLL_Expansion2/CvMinorCivAI.cpp [hunk 2]
```diff
@@ -3424,6 +3481,11 @@ bool CvMinorCivQuest::DoFinishQuest()
 		strMessage = Localization::Lookup("TXT_KEY_NOTIFICATION_QUEST_GET_CITY_COMPLETE");
 		strSummary = Localization::Lookup("TXT_KEY_NOTIFICATION_SUMMARY_GET_CITY_COMPLETE");
 	}
+	else if (m_eType == MINOR_CIV_QUEST_UNIT_GET_CITY_OCC)
+	{
+	strMessage = Localization::Lookup("TXT_KEY_NOTIFICATION_QUEST_GET_CITY_OCC_COMPLETE");
+	strSummary = Localization::Lookup("TXT_KEY_NOTIFICATION_SUMMARY_GET_CITY_OCC_COMPLETE");
```
Comment (IanE9) [inline CvGameCoreDLL_Expansion2/CvMinorCivAI.cpp:3487]: Loose indentation here.

**Review by IanE9 — CHANGES_REQUESTED** [review]

**Comment by RecursiveVision** [comment]
This would be more effectively accomplished by altering how the quest works for humans in OCC games. The text is fine, but adding a new quest is clunky.

**Pull request closed without merging**